iam/api/ServiceAccountKeys/ServiceAccountKeysCreate.cs

```diff
@@ -34,8 +34,16 @@ public static ServiceAccountKey CreateKey(string serviceAccountEmail)
             new CreateServiceAccountKeyRequest(),
             "projects/-/serviceAccounts/" + serviceAccountEmail)
             .Execute();
-        Console.WriteLine("Created key: " + key.Name);
+
+        // The PrivateKeyData field contains the base64-encoded service account key
+        // in JSON format.
+        // TODO(Developer): Save the below key (jsonKeyFile) to a secure location.
+        //  You cannot download it later.
+        byte[] valueBytes = System.Convert.FromBase64String(key.PrivateKeyData);
+        string jsonKeyContent = Encoding.UTF8.GetString(valueBytes);
```

Comment:
This is not building because you are missing an import statent at the top of the file (alphabetically):

```
using System.Text;
```